etc/taskcluster/decision_task.py

```diff
@@ -762,7 +765,7 @@ def linux_build_task(name, *, build_env=build_env):
         .with_index_and_artifacts_expire_in(build_artifacts_expire_in)
         .with_max_run_time_minutes(60)
         .with_dockerfile(dockerfile_path("build"))
-        .with_env(**build_env, **unix_build_env, **linux_build_env)
+        .with_env(**build_env, **unix_build_env, **linux_env, **linux_build_env)
```

Comment:
Since this function calls `linux_task()` there is no need to repeat `linux_env` here.